src/oneapi/oneapi.pyx

```diff
@@ -48,27 +48,41 @@ cdef class sycl_execution_context:
 # thread-local storage
 from threading import local as threading_local
 _tls = threading_local()
-if getattr(_tls, 'initialized', None) is None:
+
+def _is_tls_initialized():
+    return (not getattr(_tls, 'initialized', None) is None) and (_tls.initialized == True)
```

Comment:
It is a valid Python, but `getattr(_tls, 'initialized', None) is not None` reads better.

Reply:
done